### PR TITLE
With Python 3 Header value should be string

### PR DESCRIPTION
**Pull request by Natim**
AssertionError: Header value b'30' is not a string in ('Retry-After', b'30')

### CODE REVIEW

File: cliquet/utils.py [hunk 1]
```diff
@@ -208,3 +209,23 @@ def build_response(response, request):
...
     return dict_obj
+
+
+def encode_header(value, encoding='utf-8'):
+    """Make sure the value is of type ``str`` in both PY2 and PY3."""
+    value_type = type(value)
+    if value_type != str:
+        # Test for Python3
+        if value_type == six.binary_type:  # pragma: no cover
+            value = value.decode(encoding)  # pragma: no cover
```
Comment (leplatrem) [inline cliquet/utils.py]: both pragmas are required ? (I would have thought that only the one for ``if`` is necessary)

File: cliquet/utils.py [hunk 2]
```diff
@@ -208,3 +209,23 @@ def build_response(response, request):
...
+        # Test for Python2
+        elif value_type == six.text_type:  # pragma: no cover
+            value = value.encode(encoding)  # pragma: no cover
+    return value
+
+
+def decode_header(value, encoding='utf-8'):
+    """Makes sure the header is an unicode string."""
```
Comment (leplatrem) [inline cliquet/utils.py]: nit: Makes vs Make ?

File: cliquet/utils.py (hunk 1 above)
Comment (Natim) [inline cliquet/utils.py]: I will try :)

File: cliquet/tests/test_utils.py [hunk 3]
```diff
@@ -101,3 +105,50 @@ def test_current_service_returns_none_for_unexisting_patterns(self):
         request.registry.cornice_services = {}
 
         self.assertEqual(current_service(request), None)
+
+
+class EncodeHeaderTest(unittest.TestCase):
+
+    def test_giving_string_return_a_string(self):
```
Comment (leplatrem) [inline cliquet/tests/test_utils.py]: nit: in other test cases, the formulation is something like ``test_returns_blah_if_foo``

**Comment by leplatrem** [comment]
r+

**Pull request merged**